plugins/TunePMuonProducer.cc

```diff
@@ -0,0 +1,47 @@
+#include "DataFormats/PatCandidates/interface/Muon.h"
```

Comment:
Ok, I will merge it today.

--
                          Shervin
Da: jec429
Inviato: martedì 5 maggio 2015 22:41
A: UMN-CMS/cms-WR
Rispondi a: UMN-CMS/cms-WR
Cc: Shervin Nourbakhsh
Oggetto: Re: [cms-WR] Muon tuneP producer (#2)



In plugins/TunePMuonProducer.cc<https://github.com/UMN-CMS/cms-WR/pull/2#discussion_r29710476>:

> @@ -0,0 +1,47 @@
> +#include "DataFormats/PatCandidates/interface/Muon.h"


I am not just selecting some muons, instead I am replacing the muon p4 with the tuneP track information. This way is easier to calculate the masses and other quantities.

—
Reply to this email directly or view it on GitHub<https://github.com/UMN-CMS/cms-WR/pull/2/files#r29710476>.